commands/run.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/buildpack/pack/logging"
 )
 
-func Run(logger logging.Logger, config *config.Config, packClient *pack.Client) *cobra.Command {
+func Run(logger logging.Logger, cfg config.Config, packClient *pack.Client) *cobra.Command {
```

Comment:
Looks like we're missing passing the additional mirrors here too (and tests, likely) (@ameyer-pivotal)

Reply:
Given that `pack run` can't be executed against a registry and given that users can't supply the image name to `pack run` selecting a mirror based on the registry in our autogenerated name (always implicitly dockerhub) feels a little awkward

Reply:
Agreed